### PR TITLE
AOM-158: Confirm before stopping app

### PR DESCRIPTION
**Pull request by patlub**
## JIRA TICKET NAME:

[AOM-158: Confirm before stopping app](https://issues.openmrs.org/browse/AOM-158)

### SUMMARY:

When a user wants to stop an app, they should be asked to confirm if they really want to

### CODE REVIEW

File: app/js/components/manageApps/Addon.jsx [hunk 1]
```diff
@@ -181,51 +181,59 @@ class Addon extends Component {
...
-        });
-        this.fetchAddon();
-      }).catch((error) => {
-        error.response.status === 401 ? location.href = `${location.href.substr(0, location.href.indexOf(location.href.split('/')[4]))}login.htm` : null;
+          starting: true
+        }):
         this.setState({
-          stopping: false,
-          starting: false,
+          stopping: true
```
Comment (Charpell) [inline app/js/components/manageApps/Addon.jsx:197]: Address indentations and remove double carriage returns like this one

File: app/js/components/manageApps/Addon.jsx [hunk 2]
```diff
@@ -181,51 +181,59 @@ class Addon extends Component {
...
+            messageBody: response.data.action == 'START' ?
+              `${moduleName} module has started`
+              :
+              `${moduleName} module has stopped`,
+            messageType: 'success',
+            showMessage: true,
+          });
+          this.fetchAddon();
+        }).catch((error) => {
+          error.response.status === 401 ? location.href = `${location.href.substr(0, location.href.indexOf(location.href.split('/')[4]))}login.htm` : null;
```
Comment (justmesam) [inline app/js/components/manageApps/Addon.jsx]: this line looks too long I guess you could split it.

**Comment by justmesam** [comment]
Except for the comments, it looks good to me.

**Review by aenkya — COMMENTED** [review]
LGTM

File: app/js/components/manageApps/Addon.jsx (hunk 1 above)
Comment (patlub) [reply app/js/components/manageApps/Addon.jsx:197]: The indention looks right to me. What do you mean by double carriage returns?

File: app/js/components/manageApps/Addon.jsx (hunk 2 above)
Comment (patlub) [reply app/js/components/manageApps/Addon.jsx]: Done

File: app/js/components/manageApps/Addon.jsx (hunk 1 above)
Comment (Charpell) [reply app/js/components/manageApps/Addon.jsx:197]: Double new line

Comment (patlub) [reply app/js/components/manageApps/Addon.jsx:197]: There are no double new lines.

**Pull request merged**